src/ReactNativeSVG.web.ts

```diff
@@ -61,6 +60,9 @@ interface BaseProps {
   style: Iterable<{}>;
 }
 
+const getHasTouchableProperty = (props: BaseProps) =>
```

Comment:
Could you change `getHasTouchableProperty` to just `hasTouchableProperty`? `getHas` sounds weird to me, but no strong opinion on that.

Reply:
@WoLewicki Done ✅